cli.js

```diff
@@ -247,6 +247,9 @@ const autotoken = (url, doctypes) => {
  * ouput: https://moncozy.mycozy.cloud
  */
 const parseCozyURL = stringUrl => {
+  if (!stringUrl.startsWith('http')) {
+    throw new Error('Cozy url must start with `http`')
```

Comment:
Wouldn't be even better to automatically add `https://` if it's missing? This way, we keep the same behaviour of our login pages where you can skip the scheme if https, e.g. "toto.mycozy.cloud", or be explicit if it's local: "http://toto.cozy.localhost:8080"

Reply:
pas bête ! Mais ça peut être `http` ou `https` donc... ?

Reply:
Si le scheme est explicite, on l'utilise (donc http ou https), sinon on ajoute `https` par défaut:
```
if (!stringUrl.startsWith('http')) {
  stringUrl += 'https://'
}  
```

Reply:
avec ça j'aurai le même plantage (qui n'est pas un message très clair) dans mon cas : instance en local (donc http) et oublie de le spécifier dans l'url

Reply:
Ok, j'avais pas dans la description de la PR  que c'était ton problème initial (mais etait-ce renseigné? ;) ). 
Laissons comme ça du coup, ça améliore déjà l'existant :+1:

Reply:
non je l'ai ajouté après vu que c'était pas clair :p